internal/commands/config_experimental.go

```diff
@@ -33,6 +34,11 @@ func ConfigExperimental(logger logging.Logger, cfg config.Config, cfgPath string
 					return errors.Wrapf(err, "invalid value %s provided", style.Symbol(args[0]))
 				}
 				cfg.Experimental = val
+				if cfg.Experimental {
+					cfg.LayoutRepositoryDir = filepath.Join(filepath.Dir(cfgPath), "layout-repo")
+				} else {
+					cfg.LayoutRepositoryDir = ""
+				}
```

Comment:
This feels a little odd to me. Are we sure we want the default layout dir in the config dir? 

Also, it looks like this isn't actually configurable right? It's either the default or nothing.

Reply:
Yeah, for now, because the feature is experimental, I didn't want to do a lot of work giving the user a configuration option. basically, the idea is:

- We need a place to save the `run-image` on disk, maybe the user doesn't care, that's why I am saving them in the pack home directory
- I am planning to document this behavior in the docs so users can be aware of it.

The other alternative is, I could work on the user experience for configuring and customizing this kind of thing, but, I was planing to do it after shipping at least something the users can play with

Reply:
Actually, I can see pack already save some data in the same folder, for example.

```=bash
➜  > tree ~/.pack -L 1
/Users/jbustamante/.pack
├── completion.zsh
├── config.toml
├── download-cache
├── layout-repo
├── registry-a932275bd19c2d9e1b88fa06698fd2f5427a363d25bf87fa500691c373089381
├── registry2908354347
└── registry57772754

6 directories, 2 files
```

Those `registry-*` folders are already saving data there

Reply:
Is it just metadata that goes in this dir? Or does it actually store large files?

Reply:
No, it will save the `run-image` in OCI layout format, it could have all the blobs depending on the user set the `sparse` flag or not. But it could have large files, for sure